model/task/task.go

```diff
@@ -836,6 +962,18 @@ func (t *Task) Insert() error {
 func (t *Task) Archive() error {
 	var update bson.M
 
+	if t.DisplayOnly {
```

Comment:
Could these execution tasks already be archived by some other mechanism, or is the only way to archive an execution task through its display task?